app/views/_masthead.html.erb

```diff
@@ -28,8 +28,8 @@
     <div class="collapse navbar-collapse pull-right" id="top-navbar-collapse" role="navigation">
       <ul class="nav navbar-nav navbar-right">
         <li><%= link_to t(:'sufia.controls.about').upcase, sufia.about_path, :'data-no-turbolink'=>"true" %></li>
-        <li><%= link_to t(:'sufia.controls.help').upcase, sufia.help_path %></li>
-        <li><%= mail_to Sufia.config.contact_email, t(:'sufia.controls.contact').upcase %></li>
+        <li><%= link_to "FAQ", faq_path %></li>
+        <li><%= link_to t(:'sufia.controls.contact').upcase, sufia.contact_path %></li>
```

Comment:
I'd upcase in the CSS, if we want all header links to be upcase (which incidentally doesn't actually match http://chemheritage.org), just use CSS `text-transform: uppercase`